docs/core/tools/sdk-errors/netsdk1071.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "NETSDK1071: A PackageReference to '{0}' specified a Version of `{1}`. Specifying the version of this package is not recommended. For more information, see https://aka.ms/sdkimplicitrefs"
+description: How to resolve the issue of a PackageReference to a metapackage included with the framework with a version.
+author: Forgind
+ms.topic: warning-reference
+ms.date: 10/09/2020
+f1_keywords:
+- NETSDK1071
+---
+# NETSDK1071: Explicitly versioned PackageReference to a metapackage that would be included with the framework. 
+
+**This article applies to:** ✔️ .NET 5.0.100 SDK and later versions
+
+When the .NET SDK issues warning NETSDK1071, it suggests there may be a version conflict in the future between the version of a metapackage specified in a PackageReference and the version of that metapackage as implicitly referenced via a TargetFramework property:
+```xml
+<PropertyGroup>
+  <TargetFramework>netcoreapp3.1</TargetFramework>
+</PropertyGroup>
+```
+
+Since the TargetFramework automatically brings in a version of the metapackage, the versions will conflict should they ever differ.
+
+To resolve this:
+1. Consider, when targeting .NET Core or .NET Standard, avoiding explicit references to `Microsoft.NETCore.App` or `NETStandard.Library` in your project file.
+2. If you need a specific version of the runtime when targeting .NET Core, use the `<RuntimeFrameworkVersion>`property instead of referencing the metapackage directly. As an example, this could happen if you're using [self-contained deployments](https://docs.microsoft.com/dotnet/core/deploying/#publish-self-contained) and need a specific patch of the 1.0.0 LTS runtime.
```

Comment:
```suggestion
2. If you need a specific version of the runtime when targeting .NET Core, use the `<RuntimeFrameworkVersion>`property instead of referencing the metapackage directly. As an example, this could happen if you're using [self-contained deployments](../../../dotnet/core/deploying/#publish-self-contained) and need a specific patch of the 1.0.0 LTS runtime.
```

Reply:
Doesn't this make it inaccurate if it moves?

Reply:
Relative links to the markdown files are generally preferred because the build validates them. Yes it's more work if a file moves, but that doesn't happen very frequently.